### PR TITLE
Bump UCX to 1.9

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

File: U/UCX/build_tarballs.jl [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
 
 name = "UCX"
-version = v"1.7.0"
+version = v"1.9.0"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://github.com/openucx/ucx/releases/download/v1.7.0/ucx-1.7.0.tar.gz",
-                  "6ab81ee187bfd554fe7e549da93a11bfac420df87d99ee61ffab7bb19bdd3371"),
+    ArchiveSource("https://github.com/openucx/ucx/releases/download/v1.9.0/ucx-1.9.0.tar.gz",
```
Comment (giordano) [inline U/UCX/build_tarballs.jl:10]:
```suggestion
    ArchiveSource("https://github.com/openucx/ucx/releases/download/v$(version)/ucx-$(version).tar.gz",
```
:smiley:

**Comment by giordano** [comment]
PowerPC doesn't like you :stuck_out_tongue_winking_eye:

**Comment by vchuravy** [comment]
On PPC:

```
[04:09:30] Making all in profiling
[04:09:30] make[3]: Entering directory '/workspace/srcdir/ucx-1.9.0/test/apps/profiling'
[04:09:30] cc -DHAVE_CONFIG_H -I. -I../../..  -DCPU_FLAGS="" -I/workspace/srcdir/ucx-1.9.0/src -I/workspace/srcdir/ucx-1.9.0 -I/workspace/srcdir/ucx-1.9.0/src  -O3 -g -Wall -Werror -fno-omit-frame-pointer -funwind-tables -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-label -Wno-long-long -Wno-endif-labels -Wno-sign-compare -Wno-multichar -Wno-deprecated-declarations -Winvalid-pch -Wno-pointer-sign -Werror-implicit-function-declaration -Wno-format-zero-length -Wnested-externs -Wshadow  -MT ucx_profiling-ucx_profiling.o -MD -MP -MF .deps/ucx_profiling-ucx_profiling.Tpo -c -o ucx_profiling-ucx_profiling.o `test -f 'ucx_profiling.c' || echo './'`ucx_profiling.c
[04:09:30] mv -f .deps/ucx_profiling-ucx_profiling.Tpo .deps/ucx_profiling-ucx_profiling.Po
[04:09:30] /bin/sh ../../../libtool  --tag=CC   --mode=link cc -O3 -g -Wall -Werror -fno-omit-frame-pointer -funwind-tables -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-label -Wno-long-long -Wno-endif-labels -Wno-sign-compare -Wno-multichar -Wno-deprecated-declarations -Winvalid-pch -Wno-pointer-sign -Werror-implicit-function-declaration -Wno-format-zero-length -Wnested-externs -Wshadow  -lm  -o ucx_profiling ucx_profiling-ucx_profiling.o ../../../src/ucs/libucs.la -lpthread -lrt -lrt  -ldl
[04:09:30] libtool: link: cc -O3 -g -Wall -Werror -fno-omit-frame-pointer -funwind-tables -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-label -Wno-long-long -Wno-endif-labels -Wno-sign-compare -Wno-multichar -Wno-deprecated-declarations -Winvalid-pch -Wno-pointer-sign -Werror-implicit-function-declaration -Wno-format-zero-length -Wnested-externs -Wshadow -o ucx_profiling ucx_profiling-ucx_profiling.o  ../../../src/ucs/.libs/libucs.a /workspace/destdir/lib/libnuma.so -lm /workspace/srcdir/ucx-1.9.0/src/ucm/.libs/libucm.a -lpthread -lrt -ldl
[04:09:30] /workspace/srcdir/ucx-1.9.0/src/ucm/.libs/libucm.a(libucm_la-event.o): In function `ucm_event_install':
[04:09:30] /workspace/srcdir/ucx-1.9.0/src/ucm/event/event.c:509: undefined reference to `ucs_load_modules'
[04:09:30] collect2: error: ld returned 1 exit status
[04:09:30] make[3]: *** [Makefile:480: ucx_profiling] Error 1
```

On x86_64:
```
[04:09:31] Making all in profiling
[04:09:31] make[3]: Entering directory '/workspace/srcdir/ucx-1.9.0/test/apps/profiling'
[04:09:31] cc -DHAVE_CONFIG_H -I. -I../../..  -DCPU_FLAGS="" -I/workspace/srcdir/ucx-1.9.0/src -I/workspace/srcdir/ucx-1.9.0 -I/workspace/srcdir/ucx-1.9.0/src  -O3 -g -Wall -Werror -fno-omit-frame-pointer -funwind-tables -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-label -Wno-long-long -Wno-endif-labels -Wno-sign-compare -Wno-multichar -Wno-deprecated-declarations -Winvalid-pch -Wno-pointer-sign -Werror-implicit-function-declaration -Wno-format-zero-length -Wnested-externs -Wshadow  -MT ucx_profiling-ucx_profiling.o -MD -MP -MF .deps/ucx_profiling-ucx_profiling.Tpo -c -o ucx_profiling-ucx_profiling.o `test -f 'ucx_profiling.c' || echo './'`ucx_profiling.c
[04:09:31] mv -f .deps/ucx_profiling-ucx_profiling.Tpo .deps/ucx_profiling-ucx_profiling.Po
[04:09:31] /bin/sh ../../../libtool  --tag=CC   --mode=link cc -O3 -g -Wall -Werror -fno-omit-frame-pointer -funwind-tables -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-label -Wno-long-long -Wno-endif-labels -Wno-sign-compare -Wno-multichar -Wno-deprecated-declarations -Winvalid-pch -Wno-pointer-sign -Werror-implicit-function-declaration -Wno-format-zero-length -Wnested-externs -Wshadow  -lm  -o ucx_profiling ucx_profiling-ucx_profiling.o ../../../src/ucs/libucs.la -lpthread -lrt -lrt  -ldl
[04:09:31] libtool: link: cc -O3 -g -Wall -Werror -fno-omit-frame-pointer -funwind-tables -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-label -Wno-long-long -Wno-endif-labels -Wno-sign-compare -Wno-multichar -Wno-deprecated-declarations -Winvalid-pch -Wno-pointer-sign -Werror-implicit-function-declaration -Wno-format-zero-length -Wnested-externs -Wshadow -o .libs/ucx_profiling ucx_profiling-ucx_profiling.o  ../../../src/ucs/.libs/libucs.so /workspace/destdir/lib/libnuma.so -lm /workspace/srcdir/ucx-1.9.0/src/ucm/.libs/libucm.so -lpthread -lrt -ldl -Wl,-rpath -Wl,/workspace/destdir/lib
[04:09:31] make[3]: Leaving directory '/workspace/srcdir/ucx-1.9.0/test/apps/profiling'
```

and https://github.com/openucx/ucx/blob/v1.9.0/test/apps/profiling/Makefile.am

Ah:

`/workspace/srcdir/ucx-1.9.0/src/ucm/.libs/libucm.a` vs `/workspace/srcdir/ucx-1.9.0/src/ucm/.libs/libucm.so`

**Comment by vchuravy** [comment]
And https://docs.binarybuilder.org/dev/troubleshooting/#Shared-library-not-built strikes again.

**Pull request merged**